rust/gui-client/src-tauri/src/client.rs

```diff
@@ -233,13 +238,7 @@ pub enum Cmd {
     },
     Elevated,
     OpenDeepLink(DeepLink),
-    /// SmokeTest gets its own subcommand because elevating would start a new process and trash the exit code
-    ///
-    /// We could solve that by keeping the un-elevated process around, blocking on the elevated
-    /// child process, but then we'd always have an extra process hanging around.
-    ///
-    /// It's also invalid for release builds, because we build the exe as a GUI app,
-    /// so Windows won't give us a valid exit code, it'll just detach from the terminal instantly.
+    /// SmokeTest gets its own subcommand for historical reasons.
     SmokeTest,
```

Comment:
I've seen `doctor` commands on other CLIs that perform a series of self-checks. That could be a design to consider where this is a debugging command that can also be used by users.

For example, if there is a permission issue, this would likely fail